### PR TITLE
Add pingdom resource to laa-legal-advisers-api

### PR DESCRIPTION
**Pull request by said-moj**
Add pingdom resource to laa-legal-advisers-api

### CODE REVIEW

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-legal-adviser-api-production/resources/pingdom.tf [hunk 1]
```diff
@@ -0,0 +1,37 @@
+provider "pingdom" {}
+
+terraform {
```
Comment (digitalronin) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-legal-adviser-api-production/resources/pingdom.tf]: You don't need to specify the backend again - it's already set in the main.tf file

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-legal-adviser-api-staging/resources/pingdom.tf [hunk 2]
```diff
@@ -0,0 +1,21 @@
+provider "pingdom" {}
+
+terraform {
```
Comment (digitalronin) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-legal-adviser-api-staging/resources/pingdom.tf]: You don't need to specify the backend again - it's already set in the main.tf file

Comment (said-moj) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-legal-adviser-api-staging/resources/pingdom.tf]: thank you, I've updated the PR

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-legal-adviser-api-production/resources/pingdom.tf (hunk 1 above)
Comment (said-moj) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/laa-legal-adviser-api-production/resources/pingdom.tf]: thank you, I've updated the PR

**Pull request merged**